<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemTab</class>
 <widget class="QWidget" name="MemTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupCheckMem">
         <property name="title">
          <string>查看内存</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelCheckAddr">
                <property name="text">
                 <string>地址:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editCheckAddr">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btnCheckMem">
              <property name="text">
               <string>查看内存</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupModifyMem">
         <property name="title">
          <string>修改内存</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelAddr">
                <property name="text">
                 <string>地址:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editModifyAddr">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="labelData">
                <property name="text">
                 <string>数据：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editModifyData">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btnModifyMem">
              <property name="text">
               <string>修改内存</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labelBase">
           <property name="text">
            <string>基
址</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelOffset">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;偏移量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tableMem">
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelTip">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;*内存表格可以直接修改&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editCheckAddr</tabstop>
  <tabstop>btnCheckMem</tabstop>
  <tabstop>editModifyAddr</tabstop>
  <tabstop>editModifyData</tabstop>
  <tabstop>btnModifyMem</tabstop>
  <tabstop>tableMem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>onMemChicked()</slot>
 </slots>
</ui>
